### ISSUE TITLE
why use AffineChannel2d instead of standard bn?

### ISSUE THREAD

**Issue by weitaoatvison** [roytseng-tw/Detectron.pytorch#105]
Hi, I found you always use bn which is implemented in the affine.py in your code instead of standard bn   provided by official pytorch. So, I'm confused about the motivition. And in the meantime, I found in affine.py, the bn you implement is just a linear transformation without do normalize. So could you give me some help? Thanks. @roytseng-tw

**Comment by w-hc** [comment]
This seems to be a legacy of the original detectron /modelling/detector.py:81
It appears that this affine transformation used to be the solution to small batch size in bn. 
Now there is group norm.

**Comment by weitaoatvison** [comment]
@w-hc Thanks. I have read the original detectron code yesterday, it seems that you are right. But I still confused that the affine.py just do a linear transformation without do normalize, so I think it can not be equal to official bn implement.

**Comment by happyharrycn** [comment]
@weitaoatvison The implementation of detectron fixes all BN layers. In this case, BN becomes strictly equivalent to an affine transform (by using the global mean and variance).

**Comment by Rizhiy** [comment]
Note about group normalisation: if you look at official detectron results, you will see that GN is much slower than BN. Unless you only care about top results and speed doesn't matter, GN is not worth it currently.

**Comment by weitaoatvison** [comment]
@happyharrycn Thanks. I found there is a transformation when loading pretrain model, it merged the calculation of mean and variance, so all bn layer can be transformed to affine layer author used.
